create_release_notes.py

```diff
@@ -0,0 +1,114 @@
+import re
+import os
+import argparse
+from patterns import *
+from datetime import datetime
+
+
+class ReleaseNoteGenerator:
+
+    def _parse_changelog(self):
```

Comment:
Jako vůbec tomu nerozumím :laughing: 

Nešlo by přidat i testy? Že bys měl nějaké vzorové data a testoval, že se to správně rozparsuje atd?

Reply:
Pridal jsem tyto testy:
    1. Release notes are empty file
    2. Release notes are non-empty file
    3. Changelog has different headline format
Popremyslim, jestli jeste nejake jsou potreba